core/src/main/java/apoc/export/json/JsonImporter.java

```diff
@@ -174,7 +176,8 @@ private void checkConstraints(List<String> labels) {
             final String importIdName = importJsonConfig.getImportIdName();
             final String missingConstraint = labels.stream().filter(label -> 
                     StreamSupport.stream(schema.getConstraints(Label.label(label)).spliterator(), false)
-                            .noneMatch(constraint -> Iterables.contains(constraint.getPropertyKeys(), importIdName))
+                            .filter(c -> c.isConstraintType(ConstraintType.UNIQUENESS) || c.isConstraintType(ConstraintType.NODE_KEY))
```

Comment:
Should this be and condition instead of or?

Reply:
I think it is right this way, that is, I verify that there is 
either a `CREATE CONSTRAINT name ON (n:LABEL) assert (n.ID_IMPORT) IS NODE KEY` 
or a `CREATE CONSTRAINT name ON (n:LABEL) assert n.ID_IMPORT IS UNIQUE`